src/server/profiles/account/account.config.js

```diff
@@ -0,0 +1,89 @@
+const { route_args, common_args, write_args } = require('../common.arguments');
+const { read_scopes, write_scopes } = require('../common.scopes');
+const { CONFIG_KEYS, VERSIONS } = require('../../../constants');
+const resource_args = require('./account.arguments');
```

Comment:
suggest that we change these back to the actual names.  When people look at this, they might assume they may be generic arguments.

account_args?  account_resource_args?

Reply:
Right. I was thinking of having these as "resource_specific_arguments" etc, so we can avoid repetitive editing.